### DISCUSSION TITLE
There's no Docs?

### DISCUSSION THREAD

**Discussion by nonunknown** [bepu/bepuphysics2#251] [General]
Hi, I found this framework, which seems to be very promissing, but theres no documentation on how to use it? for example, a "hello world", how things works internally all API calls we have access to....

**Comment by RossNordby** [comment]
Here are some docs!
https://github.com/bepu/bepuphysics2/tree/master/Documentation

I'd recommend starting with [Getting Started](https://github.com/bepu/bepuphysics2/blob/master/Documentation/GettingStarted.md). It points to various demos in the demos application.

There is *not* a prebuilt sandcastle-style API reference, but the library does have a lot of XML documentation visible through intellisense and other browsing methods.

**Reply by nonunknown** [reply]
Yeah I saw those docs, it would be good to have a documentation explaining a MVP. I would like to integrate it into another engine, so having an example using only physics would be awesome, demos are very complex for me xD

**Reply by RossNordby** [reply]
Check the [SimpleSelfContainedDemo](https://github.com/bepu/bepuphysics2/blob/master/Demos/Demos/SimpleSelfContainedDemo.cs)! (That's linked in the getting started docs! :P)
